.github/workflows/build-status-check.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
   build_perf_project:
     name: Build ${{ vars.PROJECT_NAME }}PerfTests
-    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.0.1
+    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.1.0
```

Comment:
Need to add the `checkout-ref` and `checkout-repository` workflow inputs.
Make sure to use Velaptor as a guide.  We implemented it wrong with the CASL project.